models/ajna/arbitrum/ajna_arbitrum_schema.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+
+models:
+  - name: ajna_arbitrum_erc20_pools
+    meta:
+      blockchain: arbitrum
+      project: ajna
+      contributors: [gunboats]
+    config:
+      tags: ["arbitrum", "ajna"]
+    description: List of ERC20 pools deployed on Ajna
+    columns:
+      - &blockchain
+        name: blockchain
+        description: Blockchain that Ajna pool was deployed on
+      - &version
+        name: version
+        description: Ajna versions that is deployed on the network (9 for original L2 deployment, 10 for burning bwAJNA fix)
+      - &collateral
+        name: collateral
+        description: Collateral token address
+      - &quote
+        name: quotes
+        description: Quote token address
+      - &pool_address
+        name: pool_address
+        description: Pool address
+      - &starting_interest_rate
+        name: starting_interest_rate
+        description: Initial interest rate set by deployer
+      - &tx_hash
+        name: tx_hash
+        description: TX hash of deployment
+      - &block_time
+        name: block_time
+        description: Time of deployment
+      - &block_date
+        name: block_date
+        description: Date of deployment
+      - &block_number
+        name: block_number
+        description: Block number that pool was deployed
+    tests:
```

Comment:
two things:
- (minor) can you move `tests` up above `columns`, which is a bit more consistent with rest of spellbook
- the columns passed in should match the unique keys in the model exactly -- i see only `pool_address` in the models

Reply:
you can remove `tx_hash`